.github/workflows/benchmark.yml

```diff
@@ -31,11 +31,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node.js
-        uses: actions/setup-node@v2
+      - uses: pnpm/action-setup@v2.0.1
```

Comment:
Why `2.0.1` over `2`?

Reply:
It's 2 different actions

`actions/checkout@v2` and `pnpm/action-setup@v2.0.1` for the last one, it's the latest version available that we user everywhere and is managed by Renovate